src/loader.js

```diff
@@ -284,6 +284,85 @@ function setupModuleLoader(window) {
 
           /**
            * @ngdoc method
+           * @name angular.Module#component
+           * @module ng
+           * @param {string} name Name of the component in camel-case (i.e. myComp which will match as my-comp)
+           * @param {Object} options Component definition object, has the following properties (all optional):
+           *
+           *    - `controller` – `{(string|function()=}` – Controller fn that should be associated with
+           *      newly created scope or the name of a {@link angular.Module#controller registered
+           *      controller} if passed as a string.
+           *    - `controllerAs` – `{string=}` – An identifier name for a reference to the controller.
+           *      If present, the controller will be published to scope under the `controllerAs` name.
+           *      If not present, this will default to be the same as the component name.
+           *    - `template` – `{string=|function()=}` – html template as a string or a function that
+           *      returns an html template as a string which should be used as the contents of this component.
+           *
+           *      If `template` is a function, then it is {@link auto.$injector#invoke injected} with
```

Comment:
Why are we deviating from the "normal" DDO's `template`/`templateUrl` (which don't get injected) ?
There's probably a good reason; I just want to know it :)

Maybe we should stress it more, because more experienced users might get biten by this.